tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -150,60 +159,113 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+        ]);
+
+        fluid.tests.setPageSize(pager, sequence, 3);
+        fluid.tests.verifyToolTips(pager, sequence, 3, [
+            [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "bird"} ],
+            [ {nodeName: "b", nodeText: "fish"}, {nodeName: "b", nodeText: "dragon"} ],
+            [ {nodeName: "b", nodeText: "ant"}, {nodeName: "b", nodeText: "ant"} ]
+        ]);
+
+        fluid.tests.setPageSize(pager, sequence, 4);
+        fluid.tests.verifyToolTips(pager, sequence, 4, [
+            [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "fish"} ],
+            [ {nodeName: "b", nodeText: "camel"}, {nodeName: "b", nodeText: "ant"} ]
+        ]);
+        
+        return {
+            name: "Pager tooltip tests",
+            tests: [
+                {
+                    name: "Tooltip visibility and contents",
+                    sequence: sequence
                 }
-            });
+            ]
         };
+    };
 
-        var tooltipTest = function (location) {
-            return function (idx, linkEl) {
-                var linkId = linkEl.id;
-                var link = $(linkEl);
-                sequence.push({
-                    element: link,
-                    jQueryTrigger: "focus"
+    fluid.tests.assertVisibleTips = function (pager, sequence, message, targetIds, expectedTooltip) {
+        sequence.push({
+            event: "{trackTooltips}.events.notifyFocusChange",
+            listener: function () {
+                fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
+                    jqUnit.assertNode("When the page size is " + pager.model.pageSize + ", The contents of the tooltip should be set", expectedTooltip,
+                        $("b", tooltip));
                 });
+            }
+        });
+    };
 
-                if (link.hasClass(pager.pagerBar.options.styles.currentPage)) { // assumption that this will not change between now and then!
-                    assertVisibleTips("There shouldn't be any tooltips visible when the currentPage is focused", []);
-                } else {
-                    assertVisibleTips("Only the tooltip for page link " + (idx + 1) + ", in the " + location + " page bar is visible", [linkId]);
-                }
-                sequence.push({
-                    element: link,
-                    jQueryTrigger: "blur"
-                });
-                assertVisibleTips("There shouldn't be any tooltips visible when none of the pageLinks are focused", []);
-            };
-        };
+    /**
+     * Change the page size
+     */
+    fluid.tests.setPageSize = function (pager, sequence, pageSize) {
+        sequence.push({
+            func: function () {
+                pager.locate("pageSize").val(pageSize);
+            }
+        });
+        sequence.push({
+            element: "#rendered-ioc .flc-pager-page-size",
```

Comment:
As mentioned elsewhere you should avoid using jQuery selectors like this, as it will reduce reusability. Instead, make use of the components DOM Binder.